content/cpp/concepts/vectors/terms/back/back.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: '.back()'
+Description: 'The `back()` method in c++ provides the reference to the most last element in vector.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Vectors'
+  - 'Programming'
+  - 'Data Structures'
+  - 'Collections'
+  - 'Methods'
+  - 'Algorithms'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The C++ **`.back()`** method is responsible for accessing the last element in the vector. It allows you to modify the data or element in the vector without actually removing it from the vector. It is mostly used to access the recently element added.
+
+## Syntax
+
+```pseudo
+vectorName.back();
+```
+To use vectors, it is necessary to `#include` the `vector` library as the vector is a part of STL library.
+
+The below example shows the use of `.back()` method in c++ vectors, here `numbers` is a vector which has 5 elements in it and it displays the last element in the vector with the help of the `.back()` method, then the last element in the vector is modifies and the modified value is displayed. 
+
+## Example
+
+```cpp
+#include <iostream>
+#include<vector> 
+//This is compulsory to include vectors while using vectors.
+
+int main(){
+  std::vector<int>numbers = {10,20,40,50,60};
+  std::cout<<"The last element in the vector is: "<< numbers.back()<< std::endl;
+  numbers.back() = 80;
+  std::cout <<"The last element in the vector after modification is: "<<numbers.back() <<std::endl;
+  return 0;
+  }
+```
```

Comment:
Add a line here saying that the following is the output of the above code. You can write it as you want.